test/disabled/test-sendfd.js

```diff
@@ -50,21 +50,20 @@
 const common = require('../common');
 const assert = require('assert');
 
-const buffer = require('buffer');
 const child_process = require('child_process');
 const fs = require('fs');
 const net = require('net');
 var netBinding = process.binding('net');
 const path = require('path');
 
 var DATA = {
-  'ppid' : process.pid,
-  'ord' : 0
+  'ppid': process.pid,
+  'ord': 0
 };
 
 var SOCK_PATH = path.join(__dirname,
-                          '..',
-                          path.basename(__filename, '.js') + '.sock');
+  '..',
```

Comment:
So we either chop (put every arg in it's own line)
Or align to call `(` (as it was).